encoding/series_iterator_test.go

```diff
@@ -173,12 +175,13 @@ func assertTestSeriesIterator(
 	assert.Equal(t, series.start, iter.Start())
 	assert.Equal(t, series.end, iter.End())
 	for i := 0; i < len(series.expected); i++ {
+		fmt.Println("ITERATION", i, "OF", len(series.expected))
```

Comment:
Remove this yeah?

Reply:
I'm so glad this is the one I accidentally left in haha